src/main/java/com/denizenscript/depenizen/bukkit/properties/essentials/EssentialsPlayerProperties.java

```diff
@@ -301,6 +301,22 @@ public void adjust(Mechanism mechanism) {
             }
         }
 
+        // <--[mechanism]
+        // @object PlayerTag
+        // @name remove_essentials_home
+        // @input ElementTag(String)
+        // @plugin Depenizen, Essentials
+        // @description
+        // Removes the specified player's Essentials home.
```

Comment:
`the player's Essentials home that matches the specified name`
"the specified (x)" generally corresponds to the input parameter rather than the base object